src/Gemini/Modules/Shell/Services/LayoutItemStatePersister.cs

```diff
@@ -121,30 +127,18 @@ public bool SaveState(IShell shell, IShellView shellView, string fileName)
             {
                 return false;
             }
-            finally
-            {
-                if (stream != null)
-                {
-                    stream.Dispose();
-                }
-            }
 
             return true;
         }
 
-        Type GetTypeFromContractNameAsILayoutItem(ExportAttribute attribute)
+        private static Type GetTypeFromContractNameAsILayoutItem(ExportAttribute attribute)
         {
-            if (attribute == null)
-                return null;
-
             string typeName;
             if ((typeName = attribute.ContractName) == null)
                 return null;
 
             var type = Type.GetType(typeName);
-            if (type == null || !typeof(ILayoutItem).IsInstanceOfType(type))
-                return null;
-            return type;
+            return typeof(ILayoutItem).IsAssignableFrom(type) ? type : null;
```

Comment:
I'm pretty sure this was correct before:

1. `Type.GetType` returns null if the type isn't found.
2. `IsInstanceOfType was (I think) used correctly before, based on [these docs](https://msdn.microsoft.com/en-us/library/system.type.isinstanceoftype(v=vs.110).aspx).